applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_transonic_perturbation_potential_element.cpp

```diff
@@ -70,11 +70,11 @@ KRATOS_TEST_CASE_IN_SUITE(TransonicPerturbationPotentialFlowElementRHS, Compress
     AssignPotentialsToNormalTransonicPerturbationElement(pElement);
 
     // Compute RHS
-    Vector RHS = ZeroVector(3);
+    Vector RHS = ZeroVector(4);
 
     pElement->CalculateRightHandSide(RHS, model_part.GetProcessInfo());
 
-    std::vector<double> reference{146.2643261263345,-122.1426284341492,-24.12169769218525};
+    std::vector<double> reference{146.2643261263345,-122.1426284341492,-24.12169769218525, 0.0};
```

Comment:
Thanks for adding the tests! I think it is a good idea to add two more tests for the EquationIdVector function (one testing the normal implementation for an inlet element and one testing the extended implementation for a non inlet element). Also I would keep the tests for the normal versions of the RHS and LHS for the inlet elements.